mite_websocket/__init__.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from contextlib import asynccontextmanager
+
+import websockets
+from websockets.exceptions import WebSocketException
+
+from mite.exceptions import MiteError
+
+
+logger = logging.getLogger(__name__)
+
+
+class WebsocketError(MiteError):
+    pass
+
+
+class _WebsocketWrapper:
+    def __init__(self):
+        self.connection = None
+
+    def install(self, context):
+        context.websocket = self
+
+    def uninstall(self, context):
+        del context.websocket
+
+    async def connect(self, *args, **kwargs):
+        self.connection = await websockets.connect(*args, **kwargs)
```

Comment:
When do we disconnect? Is it worth using the connection's context manager as documented on the websocket home page to connect and disconnect automatically when entering and exiting a relevant journey?

Reply:
It's a good point @jarnfield, I'll add something to do this